config/environments/development.rb

```diff
@@ -26,7 +26,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```

Comment:
Well, I'm not convinced that you should use `:amazon` for development.

Reply:
This is now taken care of